docs/content/configuration.md

```diff
@@ -13,7 +13,7 @@ Some options are available to customize how the `db-tools:backup` command works.
 
 ### Storage directory
```

Comment:
Maybe here you can rename with

```suggestion
### Storage 
```

And explain a bit that one can implement its own Filename strategy